receiver/dotnetdiagnosticsreceiver/config.go

```diff
@@ -0,0 +1,40 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dotnetdiagnosticsreceiver
+
+import "go.opentelemetry.io/collector/config/configmodels"
+
+type Config struct {
+	configmodels.ReceiverSettings `mapstructure:",squash"`
+	// Optional directory name to save binary blobs read from the socket. This can
+	// be useful for troubleshooting/debugging. If left blank or unset, no blobs are
+	// saved.
+	BlobDir string `mapstructure:"blob_dir"`
+	// The maximum number of blob files to keep in the blob_dir, if specified. Defaults
+	// to 10 (most recent).
+	MaxBlobFiles int `mapstructure:"max_blob_files"`
+	// The process ID of the dotnet process from which to collect diagnostics.
+	PID int `mapstructure:"pid"`
+	// The number of seconds between each send from the socket (or named pipe). This
+	// value is sent unchanged to the dotnet process at startup to tell it how often
+	// to send metrics. Corresponds to the "refresh-interval" option in the
+	// `dotnet-counters` tool. Defaults to 1.
+	IntervalSec int `mapstructure:"interval_sec"`
+	// A list of counters sent to the dotnet process at startup time.
```

Comment:
Sure. I went with this language because I thought it was a little more precise: it's the initial request we make to the backend at startup time. What the backend actually sends us may or may not contain these counters depending on what's available, but maybe that's splitting hairs.